core/templates/dev/head/pages/exploration_player/LearnerLocalNav.js

```diff
@@ -20,9 +20,10 @@ oppia.constant(
 
 oppia.controller('LearnerLocalNav', [
   '$scope', '$modal', '$http', 'ExplorationPlayerService', 'alertsService',
-  'UrlInterpolationService', 'focusService', 'FLAG_EXPLORATION_URL_TEMPLATE',
-  function($scope, $modal, $http, ExplorationPlayerService, alertsService,
-    UrlInterpolationService, focusService, FLAG_EXPLORATION_URL_TEMPLATE) {
+  'UrlInterpolationService', 'FocusManagerService',
+  'FLAG_EXPLORATION_URL_TEMPLATE', function($scope, $modal, $http,
```

Comment:
Let's move the function to the next line and format it somewhat like 

![function](https://user-images.githubusercontent.com/7807557/31314013-7a68d46e-ac12-11e7-9659-982bf6f9a57a.jpeg)

Reply:
```
  'FLAG_EXPLORATION_URL_TEMPLATE', 
  function(
    $scope, $modal, $http, ExplorationPlayerService, alertsService, 
    UrlInterpolationService, FocusManagerService, FLAG_EXPLORATION_URL_TEMPLATE) 
    {
    $scope.explorationId = ExplorationPlayerService.getExplorationId();
```
Is it fine? @shubha1593

Reply:
Hi @Nix07, curly brace can remain in the same line 
`..., FLAG_EXPLORATION_URL_TEMPLATE) {'

Reply:
Then, the number of characters in a single line will exceed 80, which is not possible. I think I have to indent every line after the curly brace?

Reply:
Nope just move the last argument down with the curly brace to fix the number of characters issue.

![functionformat](https://user-images.githubusercontent.com/7807557/31314432-414e5ef4-ac1e-11e7-9293-1d8e8edfdd85.jpeg)